pyrep/objects/octree.py

```diff
@@ -1,11 +1,14 @@
-from pyrep.objects.object import Object
-from pyrep.const import ObjectType
 from pyrep.backend import sim
-from typing import List, Optional
+from typing import List, Optional, Union
+from pyrep.objects.object import Object, object_type_to_class
+from pyrep.const import ObjectType
 
 class Octree(Object):
     """An octree object."""
 
+    def __init__(self, name_or_handle: Union[str, int]):
```

Comment:
Why do we need this init? Why cant we just use the parents init?

Reply:
Hey @stepjam,
I was asking me myself the same question, and well I not that sure.
However, I think it might be something related to running multiple instances of pyrep and Coppelia and their specific handles. Because I was trying to run it without their init and it kept raising me the Erro 11 when there were multiple instances running in parallel, and after specifying this init it stopped happening.
But I mean, if it doesn't make much sense skip this and just accept the `object_type_to_class[ObjectType.OCTREE] = Octree` line and live this behind.

Reply:
OK, strange. I'll take your word on that and leave it in. Thanks